pkgs/tools/misc/tmux-sessionizer/default.nix

```diff
@@ -19,16 +19,18 @@ in rustPlatform.buildRustPackage {
   src = fetchFromGitHub {
     owner = "jrmoulton";
     repo = name;
-    rev = "v${version}";
-    hash = "sha256-8RQ67v2Cw681zikxYnq0Pb2ybh26w8mUbHKAC4TjYWA=";
+    rev = "79ab43a4087aa7e4e865cab6a181dfd24c6e7a90";
+    hash = "sha256-gzbCeNZML2ygIy/H3uT9apahqI+4hmrTwgXvcZq4Xog=";
   };
 
-  cargoHash = "sha256-ZOWoUBna8U0A/sYwXMf4Z7Vi+KqM7VinWhmtO8Q0HtU=";
+  cargoHash = "sha256-OFqxZu/It1j+MihN2Mb6yNBf90AZk7PVL82ml2jjjQo=";
 
   passthru.tests.version = testers.testVersion {
     package = tmux-sessionizer;
   };
 
+  OPENSSL_NO_VENDOR = 1;
```

Comment:
suggestion: add a comment about why this is necessary :smile:

Reply:
I did a global search for this env var and a bunch of packages had it already, so I thought this was a common fix, but I can add a comment if you like

Reply:
Not blocking on this, just a suggestion 🙂